engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -527,6 +567,33 @@ public JSONObject getLogicalOperator(@PathParam("operatorName") String operatorN
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName:.+}")
```

Comment:
you don't need the :.+ here.  I assume there will be another PR that addresses little things like this so I'll stop here until I see the PR that is supposed to be merged.